.snyk

```diff
@@ -0,0 +1,25 @@
+---
+ignore:
+  5ed8b131-b1a4-4c9f-a6b4-75eecf2396b7:
+    - "vendor/github.com/docker/distribution/registry/api/v2/urls.go":
+      reason: |
+        SSRF: The URL is static, coming from the "${DOCKERCONFIG}" file, thus
+        while the URL is configurable it's not dynamic.
+      expires: "2025-01-01"
```

Comment:
nit pick: I don't think we should choose a date that falls on a non-working day.

Reply:
Sure! Good point.